README.md

```diff
@@ -0,0 +1,125 @@
+<!-- Copyright 2018-2020 Cargill Incorporated
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. -->
+
+# CanopyJS
+
+CanopyJS is a library for building Canopy applications. A Canopy application is
+a React app that is capable of dynamically loading in saplings, which are UI
```

Comment:
should `saplings` be capitalized?

Reply:
I think a link to [SaplingJS](https://github.com/cargill/saplingjs) would be helpful for readers to connect CanopyJS and SaplingJS